schematic/models/validate_attribute.py

```diff
@@ -30,6 +30,40 @@
 MessageLevelType = Literal["warning", "error"]
 ScopeTypes = Literal["set", "value"]
 
+ValueValidationOutput = tuple[pd.Series, pd.Series, pd.Series]
+"""
+This is the result of doing cross validation when of type value.
+This is a tuple made of three pandas Series objects
+
+The first Series are the missing_values: values that are present in the source manifest,
+    but not present in the target manifest
+The second Series are the duplicated_values: values that duplicated in the concatenated
+    target column, and also present in the source manifest column
+The third Series are the repeat_values: values that are repeated between the manifest
+    column and concatenated target column
+"""
+
+
+@dataclass
+class SetValidationOutput:
+    """
+    This is the result of doing cross validation when of type set.
+    target_manifests: All manifests the input manifest was checked against
```

Comment:
Could you add these to an Attributes section of the docstring:
https://google.github.io/styleguide/pyguide.html#384-classes